Bio/Blast/ParseBlastTable.py

```diff
@@ -14,6 +14,8 @@
 
 
 class BlastTableEntry(object):
+    """Store the Blast Table Entry, the field values from the table."""
```

Comment:
Instead of `Store the Blast Table Entry, ...`, `Container for Blast table entry data` seems like a better fit.

Reply:
Agree, amended.